### ISSUE TITLE
Extra content with <ul> and <li> breaks rendering

### ISSUE THREAD

**Issue by darrenjennings** [vuejs/rollup-plugin-vue#162]
### Expected behavior
This should work ok inside render function.
```jsx
      <div>
          <h1>This h1 shouldn't break anything</h1>
          <ul>
            {this.list.map((value) => {
              return <li>{value}</li>;
            })}
          </ul>
      </div>
```

Also this should work also:
```jsx
      <div>
          <ul>
            <li>First item</li>
            {this.list.map((value) => {
              return <li>{value}</li>;
            })}
          </ul>
      </div>
```

### Actual behavior
Nothing is rendered. If you remove everything but the ul and .map iteration then it works.

### Steps to reproduce the behavior
Check out repo https://github.com/darrenjennings/example-functional-rollup-plugin-vue forked from znck/example-functional-rollup-plugin-vue specifically this file: https://github.com/darrenjennings/example-functional-rollup-plugin-vue/blob/master/src/Bye.vue

**Comment by znck** [comment]
Thanks for repro. I'm looking into it.

**Comment by znck** [comment]
The vue plugin now converts script content to import statements. This is fixed with v4.0.0.

Related #153
